docs/javascript.md

```diff
@@ -124,6 +124,166 @@ if (isLoggedIn) {
     { premiumStatus: 'not_available' };
 }
 ```
+- ### Server Interations.
+> You should extract similar part of the code to the separate functions/classes
+> Instead of
+
+```javascript
+// user.service.js
+
+class UserService {
+  getUsers() {
+    return axios.get('http://localhost:4200/api/users',
+      {headers: {
+        'Authorization': localStorage.getItem('token'),
+      }}
+    )
+  }
+  updateUser(user) {
+    return axios.put('http://localhost:4200/api/users',
+      user,
+      {headers: {
+        'Authorization': localStorage.getItem('token'),
+      }}
+    )
+  }
+}
+```
+> Better to use
+
+Best way to handle that is to create `HttpService` class which will be responsible for all basic things in all services

```

Comment:
The best way to handle that is to create `HttpService` class which will be responsible for all the basic things in all servicesWrite.